content/cluster/_index.md

```diff
@@ -79,3 +79,16 @@ EOF
 
 oc get pods -n openshift-monitoring -o wide
 ```
+
+## Setting the control-plane nodes as NoSchedulable
+
+This point is optional as you may want to keep your control-plane nodes schedulable (for example if you are running >=OCPv4), otherwise, you may want to mark them as NoSchedulable:
```

Comment:
This point is optional as you may want to keep your control-plane nodes schedulable, otherwise, you may want to add a `NoSchedule` taint as:

Reply:
Also maybe adding a link to the k8s docs on what a taint is https://kubernetes.io/docs/concepts/configuration/taint-and-toleration

Reply:
Adding in next commit.